src/object/schema_manager.c

```diff
@@ -15749,24 +15752,67 @@ sm_truncate_using_destroy_heap (MOP class_mop)
   insts_hfid = sm_ch_heap ((MOBJ) class_);
   assert (!HFID_IS_NULL (insts_hfid));
 
+  lob_attrid_arr = (int *) malloc (sizeof (int) * class_->att_count);
+
+  for (attr = class_->ordered_attributes; attr; attr = attr->order_link)
+    {
+      if (attr->type->id == DB_TYPE_BLOB || attr->type->id == DB_TYPE_CLOB)
+	{
+	  lob_attrid_arr[attrid_arr_length++] = attr->id;
+	}
+    }
```

Comment:
LOB 타입이 존재하는 경우에만 malloc() 수행

```suggestion
  for (attr = class_->ordered_attributes; attr; attr = attr->order_link)
    {
      if (attr->type->id == DB_TYPE_BLOB || attr->type->id == DB_TYPE_CLOB)
	{
	  if (lob_attrid_arr == NULL)
	    {
	      lob_attrid_arr = (int *) malloc (sizeof (int) * class_->att_count);
	    }
	     
	  lob_attrid_arr[attrid_arr_length++] = attr->id;
	}
    }
```

Reply:
lob 타입이 있는 경우, lob 타입 개수만큼만 배열을 생성하도록 변경하였습니다.